CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 
 ### Changes
 
+* [#5949](https://github.com/rubocop-hq/rubocop/issues/5949): Lint/EnsureReturn cop - why ensure must not return. ([@andriymosin][])
```

Comment:
This is not a feature change to end users, addition to CHANGELOG.md is unnecessary.

Reply:
You right, I wasn't sure about that as well. Will remove this